pkg/identity/extensions_test.go

```diff
@@ -0,0 +1,261 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package identity
+
+import (
+	"crypto"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/internal/testpeertls"
+	"storj.io/storj/pkg/peertls"
+)
+
+type extensionHandlerMock struct {
+	mock.Mock
```

Comment:
try using a field count and return nil instead of using `mock.Mock`, the code should end up nicer